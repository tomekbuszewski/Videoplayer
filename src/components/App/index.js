/**
 * @author tomekbuszewski
 * @since 2018-11-26
 */

import * as React from 'react';
import inject from 'react-jss';

import styles from './index.styles';

const App = () => (
  <div>Hello</div>
);

export default inject(styles)(App);
