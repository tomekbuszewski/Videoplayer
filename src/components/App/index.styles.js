import { FONT_SIZE_SMALL } from '@config/fonts';

export default {
  '@global': {
    html: {
      fontSize: FONT_SIZE_SMALL,
    },

    body: {
      fontFamily: '"Roboto Mono", "Courier New", monospaced',
    },

    '*': {
      margin: 0,
      boxSizing: 'border-box',
      padding: 0,
    },
  },
};
