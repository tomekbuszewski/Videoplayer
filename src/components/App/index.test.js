import * as React from "react";
import renderer from "react-test-renderer";

import App from "./";

describe("App tests", () => {
  it ("should match snapshot", () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
