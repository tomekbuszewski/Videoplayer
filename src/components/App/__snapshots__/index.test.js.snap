// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should match snapshot 1`] = `
<div>
  Hello
</div>
`;
